packages/tx/src/eip4844Transaction.ts

```diff
@@ -0,0 +1,479 @@
+import { byteArrayEquals, toHexString } from '@chainsafe/ssz'
+import {
+  Address,
+  MAX_INTEGER,
+  bigIntToHex,
+  bigIntToUnpaddedBuffer,
+  bufferToBigInt,
+  bufferToHex,
+  computeVersionedHash,
+  ecrecover,
+  toBuffer,
+} from '@ethereumjs/util'
+import { keccak256 } from 'ethereum-cryptography/keccak'
+
+import { BaseTransaction } from './baseTransaction'
+import { kzg } from './kzg/kzg'
+import {
+  BLOB_COMMITMENT_VERSION_KZG,
+  BlobNetworkTransactionWrapper,
+  BlobTransactionType,
+  LIMIT_BLOBS_PER_TX,
+  SignedBlobTransactionType,
+} from './types'
+import { AccessLists, checkMaxInitCodeSize } from './util'
+
+import type {
+  AccessList,
+  AccessListBuffer,
+  AccessListBufferItem,
+  BlobEIP4844TxData,
+  JsonTx,
+  TxOptions,
+  TxValuesArray,
+} from './types'
+import type { Common } from '@ethereumjs/common'
+
+const TRANSACTION_TYPE = 0x05
+const TRANSACTION_TYPE_BUFFER = Buffer.from(TRANSACTION_TYPE.toString(16).padStart(2, '0'), 'hex')
+
+const validateBlobTransactionNetworkWrapper = (
```

Comment:
Can this method get a short code doc description, so: what it does?

Reply:
Update: is this method really necessary since only called once anyhow? Totally no strong opinion here though.

Reply:
We don't absolutely need it though I think it makes a nice utility method for doing format validation.  I could be persuaded either way though.

Reply:
Ok, sure, let's keep